pages/docs/reference/whatsnew1420.md

```diff
@@ -0,0 +1,348 @@
+processes in parallel. When the dependency requirements clash, the two installations of npm packages can cause errors.
+
+To resolve this issue, Kotlin 1.4.20 includes a new (experimental) option to disable these so-called "granular workspaces".
+This feature is currently available through the `YarnRootExtension` inside the `YarnPlugin` in Gradle.
+To use it, add the following snippet to your `build.gradle.kts` file:
+
+```kotlin
+rootProject.plugins.withType(YarnPlugin::class.java) {
+    rootProject.the<YarnRootExtension>().disableGranularWorkspaces()
+}
+```
+
+### New Wizard templates
+
+To give you more convenient ways to customize your project during creation, the project wizard for Kotlin comes with new 
+templates for Kotlin/JS applications:
+- **Browser Application** - a minimal Kotlin/JS Gradle project that runs in the browser.
+- **React Application** - a React app that uses the appropriate `kotlin-wrappers`.
+    It provides options to enable integrations for style-sheets, navigational components, or state containers.
+- **Node.js Application** a minimal project for running in a Node.js runtime. It comes with the option to directly
+    include the experimental `kotlinx-nodejs` package.
+    
+Learn more how to create Kotlin/JS applications from templates [here](../tutorials/javascript/setting-up.html).
+
+### Ignoring compilation errors with IR compiler
+
+> _Ignore compilation errors_ mode is [experimental](evolution/components-stability.html) in 1.4.20. It may change incompatibly in future versions.
+{:.note}
+
+The [IR compiler](js-ir-compiler.html) for Kotlin/JS comes with a new experimental mode - _compilation with errors_.
+In this mode, you can run you code even if it contains errors, for example, if you want to try certain things it when 
+the whole application is not ready yet.
+ 
+There are two tolerance policies for this mode:
+- `SEMANTIC`: the compiler will accept code which is syntactically correct, but doesn't make sense semantically,
+    such as `val x: String = 3`.
+
+- `SYNTAX`: the compiler will accept any code, even if it contains syntax errors.
+
+To allow compilation with errors, add the `-Xerror-tolerance-policy=` compiler option with one of the values listed above.
+
+Learn more about ignoring compilation errors with Kotlin/JS IR compiler [here](js-ir-compiler.html#ignoring-compilation-errors).
+
+[**Back to top**](#)
+
+## Kotlin/Native
+
+### Escape analysis
+
+> The escape analysis mechanism is [experimental](evolution/components-stability.html) in 1.4.20. It may change incompatibly in future versions.
+{:.note}
+
+Kotlin/Native receives a prototype of the new [escape analysis](https://en.wikipedia.org/wiki/Escape_analysis) mechanism.
+It improves the runtime performance by allocating certain objects on the stack instead of the heap. With this mecahnism, 
+we've got a 10% average performance increase on our benchmarks, and we continue improving it so that it speeds up the
```

Comment:
Avoid using _we_ :)

Reply:
I'd leave `our benchmarks` and `we continue improving` because:
- these are actually `our (internal) benchmarks`, not some widely known ones and not real-life projects
- `we continue improving` is what the Kotlin team (= we) works on. There are no other actors